tests/test_api.py

```diff
@@ -79,3 +82,26 @@ def test_collection_argument_with_dollar() -> None:
                 matched = True
                 assert action.ctl.ruleRemoveTargetById == 942290
                 assert action.ctl.removeVariableName == "json.flags.$notjunk"
+
```

Comment:
I forgot to add this assertion in my new test:
```suggestion
    assert matched
```

Reply:
Still missing :)

Reply:
It is on line 112. It doesn't make sense to add it to line 84, it we are setting it true in line 82, right?

Reply:
Sorry, line 82 belongs to **my** new test, where I had forgotten to add that assertion at the end. I'm just trying to smuggle it into your PR ;)

Reply:
Got it now 😄